docs/user-guide/src/capm3/installation_guide.md

```diff
@@ -0,0 +1,29 @@
+# Install Cluster-api-provider-metal3
+
+You can either use clusterctl (recommended) to install Metal³ infra provider
+or kustomize and directly install manifests yourself. Please keep in mind that
+Baremetal Operator and Ironic are decoupled from CAPM3 and will not be installed
+when the provider is initialized. As such, you need to install them yourself and you
+can do it by following the instructions in [Install Baremetal Operator](../bmo/install_baremetal_operator.md)
+and [Install Ironic](../bmo/install_ironic.md).
```

Comment:
Do these instructions already exist or we should wait for them first and then merge this?
Also, ironic installation document path looks wrong to me, should not it be (../ironic/install_ironic.md)?

Reply:
no they don't exist yet. Removed